src/key_defs_sysctl.h

```diff
@@ -28,7 +28,7 @@
 #define System_Break (Key) { HID_SYSTEM_BREAK, KEY_FLAGS | SYNTHETIC|IS_SYSCTL | HID_TYPE_OSC }
 #define System_DebuggerBreak (Key) { HID_SYSTEM_DEBUGGER_BREAK, KEY_FLAGS | SYNTHETIC|IS_SYSCTL | HID_TYPE_OSC }
 #define System_ApplicationBreak (Key) { HID_APPLICATION_BREAK, KEY_FLAGS | SYNTHETIC|IS_SYSCTL | HID_TYPE_OSC }
-#define System_ApplicationDebuggerBreak (Key) { HID_APPLICATION_DEBUGGER_BREAK | HID_TYPE_OSC }
+#define System_ApplicationDebuggerBreak (Key) { .raw = HID_APPLICATION_DEBUGGER_BREAK | HID_TYPE_OSC }
```

Comment:
gcc (version 5.2) had problems with this line on x86. I suppose that's because there is just one value passed to the brace initializer. For the compiler it is not obvious, given the fact that we are initializing a union, to which of the union members to assign the single value (keyCode or raw). Explicitly setting `.raw` as target resolves this issue. This seems not to be a problem when two values are passed to the brace initialize list.

Arduino is using a slightly older version of gcc. That's why the error  does not show up when compiled with current Arduino's avr-gcc.